runtime/contributing/docs.md

```diff
@@ -14,9 +14,17 @@ repository. These pages are not directly editable:
 - The [API reference](/api/deno/) pages are generated from type definitions in
   the Deno repository.
 - The [CLI reference](/runtime/reference/cli/) pages for each individual command
-  are generated from source files in the Deno repository.
+  are generated from source files in the Deno repository. Check
+  [_commands_reference.json](https://github.com/denoland/docs/blob/main/runtime/reference/cli/_commands_reference.json).
```

Comment:
The source is here https://github.com/denoland/deno/blob/c2abcb9a3de77d3c8ee502d12ec5a27c55e17c9c/cli/args/flags.rs#L4204-L4207 . The `_commands_reference.json` file is generated from the deno binary.